tests/docker/build_globus.sh

```diff
@@ -0,0 +1,11 @@
+# build the builder
+docker build -t irods-s3-api-builder -f irods_builder.Dockerfile .
```

Comment:
"irods-s3-api-builder" -> "irods-globus-connector-builder"

Reply:
This file was a copy from S3 API.  It is not needed or used.  I removed it.